tests/conftest.py

```
@@ -71,39 +80,53 @@ def event_loop_cleanup() -> Generator[None, None, None]:
 
         if loop and not loop.is_closed():
-            # Get all pending tasks in this loop
-            try:
-                pending = asyncio.all_tasks(loop)
-            except RuntimeError:
-                # asyncio.all_tasks() can fail if loop is from another thread
-                return
+            # Suppress asyncio exception logging during cleanup
+            # Prevents "Task was destroyed but it is pending!" warnings
+            old_exception_handler = loop.get_exception_handler()
+            loop.set_exception_handler(lambda loop, context: None)
```

Ruff (0.14.2)

Unused lambda argument: `loop`

(ARG005)